app/Http/Controllers/LinkedArtController.php

```diff
@@ -290,10 +290,27 @@ private function getDimensions($artwork): array
             '_label' => 'millimeters',
         ];
 
-        if (!empty($artwork->dimension_width)) {
+        $diameterValue = $artwork->dimensions_detail[0]['diameter_cm'] ?? 0;
+        if ($diameterValue > 0) {
             $dimensions[] = [
                 'type' => 'Dimension',
-                'value' => $artwork->dimension_width,
+                'value' => $diameterValue * 10,
```

Comment:
Why do we store the dimensions in cm if we're going to display them in mm?

Reply:
CITI gives it to us in CMs.